frontend/src/types/analytics.ts

```diff
@@ -23,6 +25,19 @@ export type Events = {
     /** the identifier of the image */
     identifier: string
   }
+  /**
+   * Description: The user clicks the CTA button to the external source to use the image
+   * Questions:
+   *   - How often do users go to the source after viewing a result?
```

Comment:
This is a bit inconsistent with the previous event, but it's better, so could you update the docstring of `CLICK_HOME_GALLERY_IMAGE` to match this?